web/packages/core/src/ruffle-player.ts

```diff
@@ -331,11 +331,19 @@ export class RufflePlayer extends HTMLElement {
                     resolve();
                 }, 200);
             });
-            this.container.style.visibility = "visible";
+            this.container.style.visibility = "";
         }
 
-        const autoplay = config.autoplay ?? AutoPlay.Auto;
-        const unmuteVisibility = config.unmuteOverlay ?? UnmuteOverlay.Visible;
+        const autoplay =
+            Object.values(Object(AutoPlay)).indexOf(config.autoplay) >= 0
```

Comment:
Can we use `.includes` instead of `.indexOf`?

Reply:
Yep, clearly better. 👍